docs/src/main/paradox/java/http/routing-dsl/directives/path-directives/ignoreTrailingSlash.md

```diff
@@ -0,0 +1,18 @@
+<a id="ignoreTrailingSlash-java"></a>
+# ignoreTrailingSlash
+
+## Description
+
+If the requested path ends with a trailing `/` character and the inner route is rejected with an empty `Rejection` list, 
+it retries it removing the trailing `/` character. Similarly, it retries adding a trailing `/` character if the original requested path doesn't end with a `/` character. 
+
+This directive will retry the inner route with a "flipped" trailing slash only if the mentioned inner route is not handled
+producing an empty `Rejection` list.
```

Comment:
Instead of "is not handled producing an empty `Rejection` list." maybe better "is rejected with an empty `Rejection` list"?